src/shaders/globe_raster.vertex.glsl

```diff
@@ -1,33 +1,90 @@
+#define HALF_PI PI / 2.0
+#define QUARTER_PI PI / 4.0
+#define RAD_TO_DEG 180.0 / PI
+#define DEG_TO_RAD PI / 180.0
+#define GLOBE_RADIUS EXTENT / PI / 2.0
```

Comment:
nit: Might be a good call to put these in the shader prelude closer to the other `PI` define we have: https://github.com/mapbox/mapbox-gl-js/blob/main/src/shaders/_prelude.glsl#L6

Reply:
Yes I think that's a good idea too :)